app/controllers/RepresentativeDanImportVATController.scala

```diff
@@ -105,4 +118,13 @@ class RepresentativeDanImportVATController @Inject()(identify: IdentifierAction,
     }
   }
 
+  private[controllers] def sameAccountNumber(dutyAccountNumber: String,
+                                             vatAccountNumber: String,
+                                             userAnswers: UserAnswers) = {
+    lazy val dutyAccountNumber: String = userAnswers.get(DefermentAccountPage).getOrElse("")
```

Comment:
it doesn't need to be lazy
```suggestion
    val dutyAccountNumber: String = userAnswers.get(DefermentAccountPage).getOrElse("")
```